### PR TITLE
update docusaurus; update metadata

### PR DESCRIPTION
**Pull request by dkarnutsch**

### CODE REVIEW

**Comment by kaufmo** [comment]
pls do not use npm anymore switch to yarn berry

**Comment by dkarnutsch** [comment]
> pls do not use npm anymore switch to yarn berry

done. I thought this was something for a different MR, but now I included it anyways.

**Review by fraxachun — APPROVED** [review]

File: .yarnrc.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+yarnPath: .yarn/releases/yarn-3.1.0.cjs
```
Comment (kaufmo) [inline .yarnrc.yml]: why not the current stable 3.2.0 release?

Comment (dkarnutsch) [reply .yarnrc.yml]: done.

**Review by kaufmo — APPROVED** [review]

**Pull request merged**